### ISSUE TITLE
STT0084: STT wav2vec2 finetuning on kabjye dilgo khentse rinpoche speech.

### ISSUE THREAD

**Issue by gangagyatso4364** [OpenPecha/stt-wav2vec2#13]
### Description
we need to train wav2vec2 model for specific speaker accent and compare the performance with the base model on test data of that particular speaker.

Training data:
[training dataset hf](https://huggingface.co/datasets/ganga4364/Dilgo-Khyentse-Rinpoche-dataset)
### Completion Criteria
A model that is capable of transcribing situ rinpoche audios accurately. 
### Implementation
1. create train val and test split of khentse rinpoche data.
2. then train the wav2vec2 model for khentse rinpoche by taking the previous checkpoint.
3. evaluate the result of model performance on test set.
4. compare the performance with the base model.
5. create a blog post documenting the results of the experiment
### subtask

- [x] Extract khentse rinpoche  speech-to-text dataset.
- [x] Preprocess the extracted speech data .
- [x] Split the dataset into training, validation, and test sets.
- [x] Set up vast ai  for testing the pipeline for fine tuning.
- [x] run prepare dataset for training  data. 
- [x] Train the model and monitor training metrics.
- [x] Evaluate the performance of fine-tuned model on the speaker-specific test data.
- [x] Compare the performance metrics of the fine-tuned model with those of the baseline model.
- [x] Document the findings and potential performance improvements based on additional speaker-specific data acquisition.

**Comment by gangagyatso4364** [comment]
the link to documentation of model trained is here:
[Documentation](https://forum.openpecha.org/t/a-custom-asr-model-to-transcribe-the-speech-of-kabjye-dilgo-khyentse-rinpoche/170)

**Comment by gangagyatso4364** [comment]
stt single speaker finetuning repo:
[repo link](https://github.com/OpenPecha/stt-wav2vec2-single-speaker.git)
